nextcord/interactions.py

```diff
@@ -359,6 +364,43 @@ async def delete_original_message(self) -> None:
             session=self._session,
         )
 
+    async def send(self, *args, **kwargs) -> Optional[Message]:
+        """|coro|
+
+        This is a fallback function for help when sending messages in
+        response to an interaction. If the response
+        :property:`InteractionResponse.is_done()` then a messahe os sent
+        to the :property:`Interaction.channel` instead.
```

Comment:
I see a few typos. Also a bit picky, but shouldn't it be this?
```rst
        This is a shorthand function for helping in sending messages in
        response to an interaction. If the response
        :property:`InteractionResponse.is_done()` then the message is sent
        via the :property:`Interaction.channel` instead.
```

Reply:
oh god i did spell a few things wrong